### PR TITLE
U/jrbogart/pg truth

### PR DESCRIPTION
**Pull request by JoanneBogart**
1. Add basic information about SQL and python module psycopg2 in new sections at the end of the  intro notebook (postgres_object_1_intro)
2. Add a new notebook explaining how to access star truth information via Postgres

### CODE REVIEW

**Comment by amit1rrr** [comment]
@JoanneBogart Sorry to intrude but I noticed this PR contains Jupyter notebook files that are hard to review on GitHub due to the JSON diffs.

You might want to checkout [ReviewNB](https://www.reviewnb.com/), a code review tool for Jupyter notebooks. 

- It's free for open source repositories
- Provides visual diffs for notebooks & ability to comment on notebook cells
- Integrates with your repositories on GitHub

jupyter_notebook_diff

**Comment by JoanneBogart** [comment]
Thanks @amit1rrr, good to know.
In this case it's not so crucial since the changes are 
1. An entirely new notebook
2. Changes to another notebook almost entirely consisting of additional, purely expository sections at the end.

**Comment by BrunoSanchez** [comment]
## Great additions
The notebooks look great. Only minor comments, just to be picky.

**For the tutorial in SQL** there is nothing to add, though I think that instead of being the last part (or appendix, fwiw) the SQL primer and `psycopg` module tutorial could potentially deserve a whole notebook or special readme file. 
I know this could be splitting things and confusing, but at the same time, they could easily be an `postgres_0_sqlprimer_tutorial` on its own.

Regarding the **postgres truth**, this is correct, and I think quite clear. Could be good to explain functions defined in the first group of code cells, just a few lines saying why and what are you defining this functions. If not, they make sense only once you actually use them, later on the tutorial.
Queries could be multi line formatted, but this is my personal taste, specially when using `JOIN` or `ORDER BY` statements.

File: tutorials/postgres_truth.ipynb [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+    "    return cond"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "trial_cond = format_cone_search('coord', '70.3', '-56.1', 30)"
```
Comment (BrunoSanchez) [inline tutorials/postgres_truth.ipynb:145]: Probably explaining what are you doing in here with some more detail.

**Pull request merged**